### PR TITLE
Feature/hiring form vaccine requirement

### PR DESCRIPTION
**Pull request by esteban-gs**
New:
-----
- Temporary field for hirer portal users to request vaccinated workers on the work order details step. The toggle, adds or removes a string to the top of the description. 

Changes:
------
- fix `tsconfigs` so:
  - cypress types don't crash with jasmine
  - absolute routes work
- new e2e tests optimized for remote e2e tests
- unit & e2e tests for new field above

### CODE REVIEW

File: src/app/online-orders/work-assignments/work-assignments.component.spec.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ import { TransportRulesService } from "../transport-rules.service";
 import { TransportProvidersService } from "../transport-providers.service";
 import { DialogService } from "primeng/dynamicdialog";
 import { MessageService } from "primeng/api";
-import { WorkAssignment } from "src/app/shared/models/work-assignment";
+import { WorkAssignment } from "../../shared/models/work-assignment";
```
Comment (chaim1221) [inline src/app/online-orders/work-assignments/work-assignments.component.spec.ts:30]: we want to move _to_ relative paths?

**Review by chaim1221 — APPROVED** [review]
if this were a company I’d be asking “why are we updating transport rules in  PR for vaccine requirements.”

the testing looks good, you should present how this works at some point.

File: cypress/utils/helpers.ts [hunk 2]
```diff
@@ -177,10 +177,10 @@ export const generateWorkAssignmentsStub = (
   };
 
   const wa2: WorkAssignment = {
+    ...wa,
     id: 2,
     transportCost: getExpectedCostForWorkerCount(2, aPaidTransportRule),
-    ...wa,
   };
-
+  console.log(JSON.stringify([wa, wa2]), "wa stub");
```
Comment (chaim1221) [inline cypress/utils/helpers.ts:184]: do we want the console.log debugging on the user side?

**Review by chaim1221 — APPROVED** [review]

**Comment by esteban-gs** [comment]
> if this were a company I’d be asking “why are we updating transport rules in PR for vaccine requirements.”
> 
> the testing looks good, you should present how this works at some point.

No changes to the transport rules. The changes dealing with transport rules are in the e2e test helper side, to stub a work order that matches some rules on time, zipcode, etc.

**Pull request merged**